pkg/plugin/state/pod.go

```diff
@@ -90,6 +91,18 @@ type PodResources[T constraints.Unsigned] struct {
 	//
 	// For pods that aren't VMs, this should be set to zero, as it has no impact.
 	Factor T
+
+	// OvercommitMillis is the amount that usage of this resource should be discounted, taken from
+	// the VirtualMachine's .Spec.Overcommit field.
+	//
+	// This field is a milli-Value -- i.e., 1000 times the (potentially fractional) value that is
+	// being expressed.
+	//
+	// As an example, setting this value equal to 1500 on all pods would allow 50% more of this
+	// resource to be scheduled (1500m ≈ 1.5 ⇒ 50% more).
+	//
+	// For pods that aren't VMs, this should be set to 1.0.
+	OvercommitMillis int64
```

Comment:
nit: I'd prefer `OvercommitMilliValue` or `MilliOvercommit` or `OvercommitMilli`, but that's just an opinion.

Reply:
Thinking about it more... WDYT about just using a `resource.Quantity` here?

Reply:
Yeah, maybe. Maybe add in a comment that we use it as general-purpose fixed-point number, because floats are complicated.

Reply:
Switched to resource.Quantity in 1f64c655dc9b3ae6a724cac9b44562e649b06709 + 11499d6ff6b46f2fe61b795e3b66a784ab59ba85